Libraries/MiscDrivers/SRAM/N01S830HA_config.h

```diff
@@ -0,0 +1,51 @@
+/******************************************************************************
+ *
+ * Copyright 2023 Analog Devices, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+#ifndef LIBRARIES_MISCDRIVERS_SRAM_N01S830HA_CONFIG_H_
+#define LIBRARIES_MISCDRIVERS_SRAM_N01S830HA_CONFIG_H_
+
+#include "N01S830HA.h"
+#include "gpio.h"
+#include "mxc_device.h"
+#include "fastspi.h"
+
+#ifndef N01S830HA_HOLD_PIN_PORT
+#define N01S830HA_HOLD_PIN_PORT MXC_GPIO0
```

Comment:
Add these defines to board.h. Maybe instead throw a build error if they're not defined elsewhere?

Reply:
https://github.com/Analog-Devices-MSDK/msdk/pull/802/commits/e6f81f632084343aeb44b1ab0cca4dd247672b54